VirtualH89/Src/h89-io.cpp

```diff
@@ -26,6 +26,16 @@ H89_IO::~H89_IO()
     debugss(ssIO, INFO, "%s\n", __FUNCTION__);
 }
 
+std::vector<DiskController *>& H89_IO::getDiskDevices()
```

Comment:
With the limited number of slots in the H89, can this just be a fixed size array, with consts for each slot? You don't need to address it in this PR, we can change it later.